web-services-guide/ent-and-web-services/client-applications.md

```diff
@@ -226,7 +226,7 @@ try {
 	$request->Users = array( $userObj );
 
 	// Request Enterprise to create user in the database.
-	$service = new AdmCreateUsersService();
+	$service = WW_DIContainer::get( WflGetNamedQueriesService::class ); // Since 10.7.0
```

Comment:
```suggestion
	$service = WW_DIContainer::get( AdmCreateUsersService::class ); // Since 10.7.0
```
The sample was `AdmCreateUsersService` otherwise the request and response data need to be updated?